src/main/java/org/opentripplanner/standalone/config/sandbox/FlexConfig.java

```diff
@@ -15,14 +13,28 @@ public class FlexConfig {
   private static final Duration MAX_TRANSFER_DURATION = Duration.ofMinutes(5);
   private static final Duration MAX_FLEX_TRIP_DURATION = Duration.ofMinutes(45);
 
-  private static final Logger LOG = LoggerFactory.getLogger(FlexConfig.class);
+  private static final Duration MAX_ACCESS_EGRESS_DURATION = Duration.ofMinutes(45);
```

Comment:
This should be inlined similar to https://github.com/opentripplanner/OpenTripPlanner/pull/4660

Reply:
Maybe I simply combine the two PRs.